packages/wrangler/src/config/config-helpers.ts

```diff
@@ -0,0 +1,36 @@
+import path from "path";
+import { findUpSync } from "find-up";
+
+/**
+ * Resolve the path to the configuration file, given the `config` and `script` optional command line arguments.
+ * `config` takes precedence, then `script`, then we just use the cwd.
+ */
+export function resolveWranglerConfigPath({
```

Comment:
This function is the meat of the PR, centralising the logic surrounding `config`, `script`, and the default option.